src/utils/filterPolicy.ts

```diff
@@ -52,35 +56,16 @@ export class FilterPolicy {
     return this.config.defaultAction;
   }
 
-  /**
-   * Reloads the policies with their latest version
-   */
-  async reloadPolicies() {
-    const policyConfig = await this.getConfig();
-
-    if (policyConfig) {
-      this.config = policyConfig;
-    }
-  }
-
-  protected async getConfig(): Promise<PolicyConfig> {
-    if (!this.configUrl) {
-      throw new Error("configUrl must be defined");
-    }
-    const configResponse = await fetch(this.configUrl); // TODO: Implement authToken
-
-    if (!configResponse.ok) {
-      throw new Error(
-        `Failed to fetch policy. Error ${
-          configResponse.status
-        }: ${await configResponse.text().catch(() => "")}`
-      );
-    }
-    const config = await configResponse.json();
-    return {
-      defaultAction: config.defaultAction,
-      owners: new Map(Object.entries(config.owners)),
-      handlers: new Map(Object.entries(config.handlers)),
-    };
+  private convertToMap(object?: {
+    [k: string]: "ACCEPT" | "DROP" | "SKIP";
```

Comment:
Could this be somehow be reused from the types generated from the json schema?

Reply:
Done!